test/external/handle-helpers

```diff
@@ -2,20 +2,19 @@
 require 'sensu-handler'
 
 class Helpers < Sensu::Handler
-
   def handle
     puts event_summary
   end
 
-  def api_request(*args)
+  def api_request(*)
     nil
   end
 
-  def stash_exists?(*args)
+  def stash_exists?(*)
```

Comment:
I'm gonna disagree with rubocop, I don't think this makes it any better and I'd argue its less readable.